cves/CVE-2014-3581.yml

```diff
@@ -66,10 +69,9 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: c164ca7383d5f204915d85a5826655d3f1557148
-     note:
-   - commit:
-     note:
+   - commit: 2fa69ec09458a7a6519eb1e960b1fee45da66143
+     note: |
+      Fixed a mod_cache NULL pointer deference when handling empty Content-Type.
 vccs:
```

Comment:
There seems to be no vccs for this vulnerability. Not sure if it was missed or intentionally left empty, but a note or some kind should be provided as this is pretty important.

Reply:
noted and added the vcc, it took a while to find. The commit was made in 2008 while the fix was implemented in 2014